### DISCUSSION TITLE
Initial Gmail Setup erroring "Unknown error occurred."

### DISCUSSION THREAD

**Discussion by The-Curtis** [moralmunky/Home-Assistant-Mail-And-Packages#839] [Q&A]
Greetings, 

I'm running HAOS. I have Mail And Packages installed via HACS. Seemingly, it has installed totally fine, but I cannot connect to my gmail account. I'm using an App Password-- All I get is an "Unknown error occurred." I can't seem to locate anything distinct in the logs, but I'm also struggling to tail logs effectively via the HA CLI.

**Comment by firstof9** [comment]
Check in `/config/home-assistant.log` for error messages.
